libs/langchain/langchain/llms/yandex.py

```diff
@@ -0,0 +1,132 @@
+from typing import Any, Dict, List, Mapping, Optional
+
+from langchain.callbacks.manager import CallbackManagerForLLMRun
+from langchain.llms.base import LLM
+from langchain.llms.utils import enforce_stop_tokens
+from langchain.load.serializable import Serializable
+from langchain.pydantic_v1 import root_validator
+from langchain.utils import get_from_dict_or_env
+
+
+class BaseYandexGPT(Serializable):
```

Comment:
If we expose BaseYandexGPT class outside, can we add a docstring for this class?
If we don't expose it, it make sense to rename in to _BaseYandexGPT.